lib/sqlalchemy/util/typing.py

```diff
@@ -64,6 +65,10 @@
 
```

Comment:
**Federico Caselli** (CaselIT) wrote:

this is only in v2

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/5504

Reply:
**Michael Bayer** (zzzeek) wrote:

Done

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/5504